core/modules/system/system.module

```diff
@@ -3222,6 +3238,35 @@ function system_breadcrumb_block($settings = array()) {
   return theme('breadcrumb', array('breadcrumb' => $crumbs));
 }
 
+/**
+ * Output the content for the powered by block.
+ *
+ * @see system_block_view()
+ */
+function system_powered_by_block($settings = array()) {
+  if (isset($settings['color']) && $settings['color'] == 'black') {
+    $primary = '#000';
+    $secondary = '#fff';
+  }
+  else {
+    $primary = '#fff';
+    $secondary = '#000';
+  }
+  backdrop_add_css('
```

Comment:
Can we add these to a .css file and load it in `backdrop_add_css()` with `'file'` instead of `'inline'`?

...either that, or use heredoc perhaps?

Reply:
@klonos Confirming that it's better to have two additional CSS files, and then choose between them based on the color setting, and then load that one, instead of adding the CSS inline? I know inline CSS doesn't get aggregated, but I thought having two additional, and very small, CSS files would be a bit pointless. Good to hear your advice though.